dbf2csv/models/dbf2csv.py

```diff
@@ -2,45 +2,61 @@
 # Copyright 2017 INGETIVE (<https://ingetive.com>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+import tempfile
 import csv
 import base64
-from odoo import models, fields, api
-from dbfpy import dbf
+import logging
 
+from odoo import models, fields, _
+from odoo.exceptions import UserError
 
-class FileDBF(models.TransientModel):
-    _name = 'dbftocsv.filedbf'
+_logger = logging.getLogger(__name__)
+
+try:
+    from dbfpy import dbf
+except ImportError as err:
+    _logger.debug(err)
+
+
+class Conversion(models.Model):
```

Comment:
Does it really have sense for this to be a normal model? I think the source dbf file is useless once converted, so IMHO this should be an `AbstractModel` (A.K.A. wizard)

Reply:
Initially it started being a wizard, but IMHO could be useful to have them stored:
* Files availables anytime, anywhere
* Relation between dbf and csv

Reply:
But I see no use case on that: you convert it, download the csv, and then you store the csv in your computer wherever you want it. If you want to download it again, you just repeat the process, isn't it?

Reply:
Ok, :pencil: